Resources/Audio/Effects/Lightning/licenses.txt

```diff
@@ -0,0 +1 @@
+LightningShock.ogg taken from Citadel Station at commit: https://github.com/Citadel-Station-13/Citadel-Station-13/commit/35a1723e98a60f375df590ca572cc90f1bb80bd5
```

Comment:
use RGA (robust generic attribution) for this. see other `attributions.yml` files for examples